ACC Algorithm/ACCAlgorithMain.c

```diff
@@ -0,0 +1,154 @@
+#include<stdio.h>
+
+//Globals
+float hostVel;
+float hostVelPrev; //Host velocity in the previous time step
+float integral = 0; //Needs to be global as it is used in the calculation of next step
+float errorPrev = -999.0; //Previous value of PID error
+float leadDistPrev = 0;
+float overshoot = 0;
+float leadTime = 0;
```

Comment:
lots of global variables to be hanging around... why not combine into a struct for the state of the system? would be nice to have different structs for different parameter sets (ie. PID params, calibrations, etc.)

ie.
```C
struct pidParams {
  float integralError;
  char prevError;
  ...
};

struct accParams {
  pidParams speedPid;
  ...
}
```

Reply:
Not sure  exactly what attributes the structs should have (what exactly should pid have and what should acc have)